question-service/route/question-route.js

```diff
@@ -1,5 +1,12 @@
 import express from "express";
 const router = express.Router();
-import * as questionController from "../controller/question-controller";
+import { getEasyQuestion } from "../controller/question-controller.js";
+import { getMediumQuestion } from "../controller/question-controller.js";
+import { getHardQuestion } from "../controller/question-controller.js";
```

Comment:
I believe you can do `import { getEasyQuestion, getMediumQuestion, getHardQuestion }`

Reply:
Understood, made the required changes.